app/assets/stylesheets/authors.css.scss

```diff
@@ -0,0 +1,13 @@
+.author {
+  clear: both;
+  .author_books {
+    list-style: none;
+    .book {
```

Comment:
I'm not sure about this rule. Maybe we should disable this check? Or set it to depth=5 ?